tests/dragonfly/connection_test.py

```diff
@@ -585,6 +585,7 @@ async def test_parser_memory_stats(df_server, async_client: aioredis.Redis):
     for i in range(0, 900):
         writer.write(b"$100\r\n" + val)
     await writer.drain()
+    await asyncio.sleep(0.1)
```

Comment:
Consider using `assert_eventually()` (currently in `cluster_test.py`, should be moved to a shareable place).
This will make this test more robust

Reply:
Ok, I made it more pythonic, so that the assert expression will actually be shown if the condition does not hold eventually.